supply/lib/supply.rb

```diff
@@ -1,3 +1,5 @@
+require 'mini_magick'
```

Comment:
I don't think we can safely add `mini_magick` as a dependency inside of one of the core `supply` functions. This will require everybody who currently runs `supply` to install imagemagick which we probably don't want as a requirement for _fastlane_ 😇

Reply:
Yes, was wondering about this. I had this change tested on my colleague's computer (who doesn't have `mini_magick` installed, and it errored out. I am not sure how to make this work. Please advise :)

Reply:
Is the removal of alpha 100% needed? 🤔 We can remove this if its not, right?

Reply:
We already have `mini_magick` in the dependencies: https://github.com/fastlane/fastlane/blob/f47f5268f899c2994b253601f1eef401b0c36615/fastlane.gemspec#L83 It's also used in frameit for example.